templates/comments.php

```diff
@@ -30,45 +30,47 @@
 <?php if (comments_open()) : ?>
   <section id="respond" class="leave-a-comment">
     <header class="group">
```

Comment:
I think this header here has to be removed/change into div

Reply:
Thanks, updated.